src/main/resources/changesets/changelog-consent-2024-04-02-rm-old-dbgap-url.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet id="changelog-consent-2024-03-28-dataset-dac-id-props.xml" author="grushton">
+    <sql>
+      -- Remove duplicate properties
+      DELETE
+      FROM dataset_property
+      WHERE property_key IN (SELECT key_id FROM dictionary d WHERE KEY = 'dbGap')
```

Comment:
Looking at what's actually in the database, does this actually need to be the following?
```suggestion
      WHERE property_key IN (SELECT key_id FROM dictionary d WHERE KEY = 'dbGAP')
```

Reply:
Nice catch, thanks for that!